src/components/ModalMovieAction/__tests__/ModalMovieAction.spec.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import store from 'store/store';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import ModalMovieAction from '../ModalMovieAction';
+import { modalType } from 'components/Modal/Modal';
+
+const renderComponent = () => {
+  return render(
+    <Provider store={store}>
+      <ModalMovieAction
+        currentModal={modalType.add} />
+    </Provider>
+  );
+};
+
+const getByName = (container, name) => container.querySelector(`[name="${name}"]`);
+const res = { json: jest.fn() };
+global.fetch = jest.fn(() => Promise.resolve(res));
```

Comment:
you overwrote global fetch function not only for this current file but for every following one as well which might cause unexpected results. such manipulations should be cleaned after test suit has been executed.

use `beforeAll` for setting it to mock function and `afterAll` to restore default behavior with https://jestjs.io/docs/jest-object#jestresetallmocks